localstack/services/apigateway/apigateway_listener.py

```diff
@@ -1,11 +1,13 @@
 import re
 import logging
 import json
+from urlparse import urljoin
```

Comment:
We're using the `six` library to stay compatible with different Python versions. Please replace this line with:
```
from six.moves.urllib_parse import urljoin
```

Reply:
👍 I was wondering why it was failing. I couldn't see much detail around the error. Cheers!